build/common.gypi

```diff
@@ -4,6 +4,7 @@
     'tizen_mobile%': 0,
     'enable_murphy%': 0,
     'use_webui_file_picker%': 0,
+    'linux_dt%': 0,
```

Comment:
I think we need to define a build parameter to distinguish default Linux build and LinuxDT.
https://github.com/crosswalk-project/crosswalk/pull/3039/files#diff-a109cb6a8605098d714d08947e753ef2R49 is an example, on the default build experimental features will be enabled, however in a release version(LinuxDT), we want these features disabled.